src/main/java/uk/gov/companieshouse/overseasentitiesapi/configuration/InterceptorConfig.java

```diff
@@ -27,14 +27,16 @@ public class InterceptorConfig implements WebMvcConfigurer {
     static final String BENEFICIAL_OWNERS_PRIVATE_DATA = "/private/**/beneficial-owners";
     static final String TRUST_DETAILS_PRIVATE_DATA = "/private/**/trusts/details";
     static final String CORPORATE_TRUSTEES_PRIVATE_DATA = "/private/**/corporate-trustees";
+    static final String TRUST_LINKS_PRIVATE_DATA = "/private/**/trusts/beneficial-owner/links";
```

Comment:
Maybe could have this as just 
```suggestion
    static final String TRUST_LINKS_PRIVATE_DATA = "/private/**/links";
```

Reply:
Would be better to be explicit about what we can match though.